var/spack/repos/builtin/packages/cppcheck/package.py

```diff
@@ -23,11 +23,9 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
-import os
-import shutil
 
 
-class Cppcheck(Package):
+class Cppcheck(MakefilePackage):
```

Comment:
not sure if it's good to wrap this into this PR, but the changes seem innocent enough